### PR TITLE
added requirements to build transit tiles.

### PR DESCRIPTION
**Pull request by gknisely**

### CODE REVIEW

File: Dockerfile-ppa [hunk 1]
```diff
@@ -3,11 +3,11 @@ MAINTAINER Grant Heffernan <grant@mapzen.com>
 
 ENV TERM vt100
 
-ENV DATA_DIR ${DATA_DIR:-"/data"}
+ENV DATA_DIR ${DATA_DIR:-"/data/valhalla"}
 ENV CONF_FILE ${CONF_FILE:-"/conf/valhalla.json"}
 
 RUN apt-get update -y && apt-get upgrade -y
-RUN apt-get install jq supervisor software-properties-common -y
+RUN apt-get install jq spatialite-bin wget unzip supervisor software-properties-common -y
```
Comment (kevinkreiser) [inline Dockerfile-ppa]: shouldnt the ppa not need these? in fact shouldnt it not need jq either. i thought the ppa knew about these deps and got them automatically

File: scripts/build_transit_tiles.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+
+# lockfile
+LOCKFILE="${DATA}/.build_transit_lock"
```
Comment (heffergm) [inline scripts/build_transit_tiles.sh]: Should this be ${DATA_DIR}?

Comment (gknisely) [reply scripts/build_transit_tiles.sh]: thanks...missed that

File: Dockerfile-ppa (hunk 1 above)
Comment (gknisely) [reply Dockerfile-ppa]: Thanks.  Updated.

**Comment by heffergm** [comment]
👍

**Pull request merged**